cmd/system-probe/modules/processmonitor.go

```diff
@@ -0,0 +1,71 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build linux
+
+package modules
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/cmd/system-probe/config"
+	sysconfigtypes "github.com/DataDog/datadog-agent/cmd/system-probe/config/types"
+	pkgconfigsetup "github.com/DataDog/datadog-agent/pkg/config/setup"
+	"github.com/DataDog/datadog-agent/pkg/process/monitor"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+
+	"github.com/DataDog/datadog-agent/cmd/system-probe/api/module"
+)
+
+func isEventStreamEnabled() bool {
+	return pkgconfigsetup.SystemProbe().GetBool("service_monitoring_config.enable_event_stream")
+}
```

Comment:
we have a module that relies on a USM configuration?

Reply:
For now, yes. The only user-facing setting that controls the usage of event stream is that config, so we have to respect it in this module. Once we do the refactor supporting dependencies of modules we should deprecate the USM config and create a common one.